common/containers/Tabs/Swap/components/PartThree.tsx

```diff
@@ -50,44 +46,41 @@ export default class PartThree extends Component<
   public render() {
     const {
       // STATE
-      originAmount,
-      originKind,
-      destinationKind,
+      origin,
+      destination,
       paymentAddress,
       orderStatus,
       destinationAddress,
       outputTx,
-      destinationAmount,
       // ACTIONS
       showNotification
     } = this.props;
 
     const SwapProgressProps = {
-      originKind,
-      destinationKind,
+      originKind: origin.id,
+      destinationKind: destination.id,
       orderStatus,
       showNotification,
       destinationAddress,
       outputTx
     };
 
     const PaymentInfoProps = {
-      originKind,
-      originAmount,
+      originKind: origin.id,
+      originAmount: origin.amount,
```

Comment:
Any reason these aren't taking in `SwapInput` objects to keep it consistent?

Reply:
No reason, just forgot to update those